zds/tutorialv2/views/contents.py

```diff
@@ -1871,13 +2001,15 @@ class ContentOfAuthor(ZdSPagingListView):
     template_name = 'tutorialv2/index.html'
     model = PublishableContent
 
+    # list: func filter, Title, Permission, Icon
     authorized_filters = OrderedDict([
-        ('public', [lambda q: q.filter(sha_public__isnull=False), _('Publiés'), True, 'tick green']),
-        ('validation', [lambda q: q.filter(sha_validation__isnull=False), _('En validation'), False, 'tick']),
-        ('beta', [lambda q: q.filter(sha_beta__isnull=False), _('En bêta'), True, 'beta']),
+        ('public', [lambda q, u: q.filter(sha_public__isnull=False, authors__pk__in=[u]), _('Publiés'), True, 'tick green']),
```

Comment:
Étant donné que tu n'as qu'un seul utilisateur :)

```suggestion
        ('public', [lambda q, u: q.filter(sha_public__isnull=False, authors__pk=u), _('Publiés'), True, 'tick green']),
```

Reply:
Ooh, je vois : https://stackoverflow.com/questions/2198635/what-does-pk-in-mean-in-django
Pardon :p